src/main/scala/io/keen/client/scala/Client.scala

```diff
@@ -361,6 +366,23 @@ trait Writer extends AccessLevel {
    */
   val writeKey: String = settings.writeKey.getOrElse(throw MissingCredential("Write key required for Writer"))
 
+  // constants
+  val MinSendInterval: Integer = 60
+  val MaxSendInterval: Integer = 3600
```

Comment:
Scratch that, `protected`. Ran into abstract method issues with threading while set to `private`.